src/components/LongTextInput.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import { useEffect } from 'react'
+import styled from 'styled-components'
+
+const StyledLongTextInput = styled.section`
+  width: 37.5rem;
+  height: 15rem;
+  border: 1px solid var(--dark-grey);
+  border-radius: 2.2px;
+  display: flex;
+  flex-direction: column;
+  padding: 1.063rem 1rem;
+
+  label {
+    font-weight: 400;
+    color: var(--dark-grey);
+  }
+
+  textarea {
+    flex: 1;
+    border: none;
+    resize: none;
+    font-weight: 400;
+    color: var(--tints-blizzard-black);
+  }
+
+  span {
+    display: block;
+    margin-left: auto;
+    font-weight: 400;
+    color: var(--dark-grey);
+  }
+
+  .max-word {
```

Comment:
Looks like there shoudl also be an outline around the whole component when there is an error. 
<img width="382" alt="Screen Shot 2023-01-28 at 5 00 02 PM" src="https://user-images.githubusercontent.com/6326891/215295025-7f794a80-6f15-46f0-88f2-540263a5f32d.png">